app/webpack/javascripts/chart.js

```diff
@@ -0,0 +1,59 @@
+import ApexCharts from 'apexcharts'
+
+let _chart = null;
+const _options = {
+  chart: {
+    type: 'bar'
+  },
+  xaxis: {
+    labels: {
+      style: {
+        fontSize: '22px'
+      }
+    }
+  },
+  yaxis: {
+    labels: {
+      style: {
+        fontSize: '22px'
+      }
+    }
+  },
+  plotOptions: {
+    bar: {
+      columnWidth: '45%',
+      distributed: true
+    }
+  },
+  dataLabels: {
+    enabled: false
+  },
+  legend: {
+    show: false
+  },
+};
+
+export const chart = {
+  render(data) {
+    const { group_labels, group_values, min, max } = data.averages;
+
+    let options = _options;
+    options.series = [{
+      name: '∅',
+      data: group_values
+    }];
+    options.xaxis.categories = group_labels;
+    options.yaxis.min = min;
+    options.yaxis.max = max;
+    // options.yaxis.formatter = (value) => {
```

Comment:
Here as well unused comment

Reply:
Shouldn't rubocop tell me this?

Reply:
That you have a comment in your JavaScript file?

Reply:
And actually, RuboCop wants you to comment. We just disabled it because we lazy